python/pyspark/ml/tests.py

```diff
@@ -929,6 +932,50 @@ def test_apply_binary_term_freqs(self):
                                    ": expected " + str(expected[i]) + ", got " + str(features[i]))
 
 
+class ALSTest(PySparkTestCase):
+
+    def test_storage_levels(self):
```

Comment:
I'm less familiar with the RDD storage API, but this also seems like it might be redundant. We already know that the setting the param takes effect on the Scala side. We don't need to check that Python successfully passes the param to the JVM since it is tested elsewhere, and I assume that the storage level transfer back to Python is also tested elsewhere. Thoughts?